jenkins-plugin/src/main/java/org/jenkinsci/plugins/pipeline/maven/reporters/JunitTestsReporter.java

```diff
@@ -159,27 +150,43 @@ public void process(@Nonnull StepContext context, @Nonnull Element mavenSpyLogsE
             return;
```

Comment:
`listener.getLogger().println(" not found, don't display org.apache.maven.plugins:maven-surefire-plugin:test results in pipeline screen.")` deserves to be adapted.